TTC.Deployment.Tests/AssumeRoleTest.cs

```diff
@@ -19,27 +15,20 @@ public class AssumeRoleTest
     {
         AwsConfiguration _awsConfiguration;
         AmazonIdentityManagementServiceClient _iamClient;
-        AmazonSecurityTokenServiceClient _securityTokenServiceClient;
         AmazonS3Client _s3Client;
-        User _user;
-        BasicAWSCredentials _userCredentials;
-        
-        Role _role;
-        string _roleName;
-
+        
         private readonly string _userName = "aws_tools_assume_role_test_user";
-        private readonly string _bucketName = "aws-tools-test-bucket-1";
+        private readonly string _bucketName = "aws-tools-test-bucket-00";
```

Comment:
We've got another branch that updates this test to use something else instead of buckets. Something that's not globally unique :)

Nick said he would merge or cherry pick that next, didn't you @Jetski5822 :)